config/config.go

```diff
@@ -99,7 +99,7 @@ type Config struct {
 	ServiceDiscoveryURL   string            `json:"serviceDiscoveryUrl"`
 	IAMServerURL          string            `json:"iamServerUrl"`
 	IAMPublicServerURL    string            `json:"iamPublicServerUrl"`
-	FileServerURL         string            `json:"fileServerUrl"`
+	SOTAServerURL         string            `json:"sotaServerUrl"`
```

Comment:
It is required by UM's. I guess we should put FileServerURL setting under smcontrolled. umcontroller will have the same option.

Reply:
Done